client/cody/src/services/LocalAppDetector.ts

```diff
@@ -0,0 +1,125 @@
+import * as vscode from 'vscode'
+
+export interface LocalProcess {
+    arch?: string
+    homeDir?: string | undefined
+    os?: string
+    isAppInstalled: boolean
+}
+
+const LOCAL_APP_LOCATIONS: { [key: string]: string[] } = {
+    // Only apply silicon is supported
```

Comment:
```suggestion
    // Only Apple silicon is supported
```

Sounds from the convo above that we support x64 as well.